### ISSUE TITLE
It does not seem to use the mixed objective optimization？

### ISSUE THREAD

**Issue by zhengge** [camel007/caffe-moon#9]
In train_val.prototxt， I do not find the mixed objective optimization.
```
layer {
 name: "euclidean-loss"
  type: "EuclideanLoss"
  bottom: "moon-fc"
  bottom: "labels"
  top: "loss"
}
```
Could you explain it? Thanks.

**Comment by bikong2** [comment]
i did not find the moon part, too.

**Comment by chaipangpang** [comment]
1.this project no relationship with the paper—“MOON”.
2.Why use the loss function —“EuclideanLoss”？It is better use “Sigmoid Cross-Entropy” or “HingeLoss”？

**Comment by BackT0TheFuture** [comment]
this repo does have noting to do with MOON loss.

@zhengge  @bikong2  @chaipangpang 

![moon](https://user-images.githubusercontent.com/10088733/45443844-7b34c280-b6f8-11e8-90a2-e698f1738e85.jpg)

did you guys make MOON loss clear?
how to calculate  src_dist ？

**Comment by klintqinami** [comment]
"Since CelebA has identical source and target distributions, we define the loss layer in (6) to weight all elements equally during backpropagation – which is equivalent to Euclidean loss between
the network output and the 40 binary attribute values."

[Page 9, MOON : A Mixed Objective Optimization Network for the Recognition of Facial Attributes]
